docs/developer-guide/operators.md

```diff
@@ -174,17 +178,76 @@ y = activation(x3, act_type, act_params)
 |15|pad_right|int|pad_left|
 |14|pad_top|int|pad_left|
 |16|pad_bottom|int|pad_top|
+|17|impl_type|int|0|
```

Comment:
impl_type has been removed

Reply:
it still in `convolution.cpp,` can it be deleted?

Reply:
no, the impl_type in convolution implementation is just for compatibility.
new models generated by ncnnoptimize will not contain this field anymore